### PR TITLE
POL-1329 Fix calculation of IOPS and Bandwith at Azure Rightsize Managed Disk

### PR DESCRIPTION
**Pull request by midir99**
### Description

This addresses the issue when calculating the IOPS and Bandwith of Premium SSD V2 disk recommendations.

### Issues Resolved

https://flexera.atlassian.net/browse/POL-1329

### Link to Example Applied Policy

Applied policy is in client environment but we received confirmation that new calculations are working well thanks to these changes, for more context please visit: https://flexera.atlassian.net/browse/POL-1329

image

### Contribution Check List

- [X] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt [hunk 1]
```diff
@@ -1060,42 +1060,52 @@ script "js_disk_rightsizing_recommendations", type: "javascript" do
...
       }
     }
-    var diskIopsAndThroughputDontFitInDiskSize = true
     var suggestedDiskSize = currentUsedGB
     var maxIops = 0
     var maxThroughput = 0
-    while (diskIopsAndThroughputDontFitInDiskSize) {
-      maxIops = suggestedDiskSize * 500
-      maxThroughput = maxIops * 0.25
+    while (true) {
```
Comment (XOmniverse) [inline cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt:1081]: This is a bad coding pattern. You should refactor this to not rely on while (true)

Instead, you should replace true with whatever condition breaks the loop. If you need this to occur after the loop iterates, you can use do while() to have the check occur after the loop.

**Review by XOmniverse — CHANGES_REQUESTED** [review]

File: cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt (hunk 1 above)
Comment (midir99) [reply cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt:1081]: Wow I did not know `do while` existed in Javascript, Implementing this change...

Comment (midir99) [reply cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt:1081]: I'm done finishing the changes and also verified them:

image

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**